guides/09_Amazon_Voice_Focus.md

```diff
@@ -283,13 +283,22 @@ A specification is used to derive a runtime _configuration_ when a transformer i
 
 ## Enabling Voice Focus with Echo Reduction
 
+> ⚠️ To use the Echo Reduction feature, you must migrate to the [Amazon Chime SDK Meetings](https://docs.aws.amazon.com/chime/latest/APIReference/API_Operations_Amazon_Chime_SDK_Meetings.html) namespace. To do so, you can follow the steps in the [Migrating to the Amazon Chime SDK Meetings namespace](https://docs.aws.amazon.com/chime/latest/dg/meeting-namespace-migration.html).
+
 Enabling Voice Focus with Echo Reduction is a two step process: 
 
-### 1. Creating a meeting with support for Echo Reduction
+### 1. Create a meeting with support for Echo Reduction using Amazon Chime SDK Meetings namespace
+
+The Amazon Chime SDK Meetings namespace uses a new service principal: meetings.chime.amazonaws.com. If you have SQS, SNS, or other IAM access policies that grant access to the service, you need to update those polices to grant access to the new service principal.
```

Comment:
Remove line 292 and 294 as they are not directly related to the Echo Reduction feature? Also, the above warning quote already has a link to the migration guide.

Reply:
Ignore this comment as builders need to grant access.